### PR TITLE
Add table helper methods

### PR DESCRIPTION
**Pull request by igitur**
Some helper methods added.

I'm not convinced about replacing `ArgumentException` with `ClosedXMLEntityNotFoundException`. I can see the use of having our own exception, but it also feels inconsistent with other places where `ArgumentException` is thrown.

### CODE REVIEW

File: ClosedXML_Tests/Excel/Tables/TablesTests.cs [hunk 1]
```diff
@@ -1009,6 +1009,17 @@ public void SavingTableWithNullDataRangeThrowsException()
...
         }
 
+        [Test]
+        public void TableNotFound()
+        {
+            using (var wb = new XLWorkbook())
+            {
+                var ws = wb.AddWorksheet("Sheet1");
+                Assert.Throws<ClosedXMLEntityNotFoundException>(() => ws.Table("dummy"));
+                Assert.Throws<ClosedXMLEntityNotFoundException>(() => wb.Table("dummy"));
```
Comment (Pankraty) [inline ClosedXML_Tests/Excel/Tables/TablesTests.cs]: Why two times?

**Comment by igitur** [comment]
Ready for review.

File: ClosedXML/Excel/IXLWorkbook.cs [hunk 2]
```diff
@@ -223,6 +223,14 @@ public interface IXLWorkbook : IDisposable
 
         XLWorkbook SetUse1904DateSystem(Boolean value);
 
+        /// <summary>
+        /// Gets the Excel table of the given name
+        /// </summary>
+        /// <param name="tableName">Name of the table to return.</param>
+        /// <param name="comparisonType">One of the enumeration values that specifies how the strings will be compared.</param>
+        /// <returns>The table with given name or </returns>
```
Comment (Pankraty) [inline ClosedXML/Excel/IXLWorkbook.cs]: The table with given name or _what_? The implementation below throws the exception if not found.

Comment (igitur) [reply ClosedXML/Excel/IXLWorkbook.cs]: ok done

**Pull request merged**